include/fi_shm.h

```diff
@@ -135,6 +135,7 @@ struct smr_region {
 struct smr_resp {
 	uint32_t	msg_id;
 	uint32_t	status;
+	uint64_t	flags;
```

Comment:
What are the flags that the target is sending back to the initiator?

Reply:
Any flags that would be needed to be set for the tx completion (so right now FI_TAGGED)